### PR TITLE
[WFLY-15044] Avoid Nullpointer (IIOP)

### PR DESCRIPTION
**Pull request by boris-unckel**
Fixes https://issues.redhat.com/browse/WFLY-15044

### CODE REVIEW

File: iiop-openjdk/src/main/java/org/wildfly/iiop/openjdk/rmi/ValueAnalysis.java [hunk 1]
```diff
@@ -173,7 +173,7 @@ protected void doAnalyze() throws RMIIIOPViolationException {
                     && wo.getReturnType() != Void.TYPE) {
                 wo = null; // Wrong return type
             }
-            if (wo != null) { // Right modifiers?
+            if (wo != null && spf != null) { // Right modifiers?
```
Comment (bstansberry) [inline iiop-openjdk/src/main/java/org/wildfly/iiop/openjdk/rmi/ValueAnalysis.java:176]: @tadamski Please review.

I suspect this isn't the right fix here and that the problem is that L177 should be

int mods = wo.getModifiers();

I don't see why the modifiers of the 'serialPersistentFields' field would be relevant at this point.

**Review by tadamski — CHANGES_REQUESTED** [review]
@bstansberry I agree, spf seems irrelevant in this place and it looks like a bug.
@boris-unckel I believe this PR should be modified according to Brian's suggestion.

**Comment by boris-unckel** [comment]
@tadamski Fixed accordingly. Could you check again and approve, please?

**Review by tadamski — APPROVED** [review]

**Comment by tadamski** [comment]
@boris-unckel Approved.

**Pull request merged**